contents/founders/cracked-manifesto.md

```diff
@@ -0,0 +1,79 @@
+---
+date: 2024-03-16
+title: A cracked manifesto
```

Comment:
This headline doesn't work, imo. Lacks context.

Reply:
I do think that having it be slightly ambiguous is a good thing - @mariusandra's feedback was that it was interesting and made him want to click. But some slightly more explanatory alternatives:

- How to hire and manage cracked engineers
- We're only hiring cracked engineers now
- Introducing the cracked engineer

Tbh I liked having manifesto in there as it signals 'hey we're putting out an alternative way of thinking about stuff', even if it is a bit self-aggrandising. Who knows, maybe in 10 years, people will hate it with the intensity they save for the agile manifesto in 2024...

Reply:
I don't feel super strongly about this and, for the newsletter at least, we can add a caption that adds some context.

Reply:
Yeah, agree the headline doesn't work. 

Why you need to hire cracked engineers

And similar would work better. Also, checking, is 'cracked' used widely-enough with this sense for people to get what we mean?

Reply:
> Why you need to hire cracked engineers

Ah yeah ok this is better. 

> Also, checking, is 'cracked' used widely-enough with this sense for people to get what we mean?

Some people yes, some people no, but the point is it's a new intriguing concept so I think that's ok?

Reply:
I think it's okay, yup. So long as nobody thinks 'cracked' is a bit weird/open to misinterpretation. It has an informal meaning of 'mentally ill.' 

I'm always down for coming up with a snappy way of referring to quite a complex idea (at WIRED, we sorta coined a name for a period of the pandemic: https://www.wired.com/story/welcome-to-the-great-reinfection/) but want to make sure we're not going to look dumb for going hard on the term 'cracked.'

Reply:
Yeah I think we're comfortable with the [risk here](https://posthog.slack.com/archives/C01FHN8DNN6/p1711081308199459) around misinterpretation, though I think for our audience, hopefully the [current](https://www.urbandictionary.com/define.php?term=Cracked) most popular usage will land!